params.go

```diff
@@ -328,6 +328,24 @@ var (
 		"Sanction length for all sanctions",
 	)
 
+	iigoTermLenghtPresident = flag.Uint(
```

Comment:
Mispelled length here.

Reply:
I should have caught that, I misspell length all the time :D